### PR TITLE
[FLINK-18798][docs-zh] Translate "Debugging Windows & Event Time" page of "Debugging & Monitoring" into Chinese

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Translate "Debugging Windows & Event Time" page of "Debugging & Monitoring" into Chinese*

## Brief change log

*Translate "Debugging Windows & Event Time" page of "Debugging & Monitoring" into Chinese*
*The page url is https://ci.apache.org/projects/flink/flink-docs-release-1.11/zh/monitoring/debugging_event_time.html*
*The markdown file is located in flink/docs/monitoring/debugging_event_time.zh.md.*

## Verifying this change

*(Please pick either of the following options)*

This change is a pure documentation work without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): **no**
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: **no**
  - The serializers: **no**
  - The runtime per-record code paths (performance sensitive): **no**
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: **no**
  - The S3 file system connector: **no**

## Documentation

  - Does this pull request introduce a new feature? **no**
  - If yes, how is the feature documented? **not applicable**

### CODE REVIEW

**Comment by RocMarshal** [comment]
Hi, @klion26  .
Could you help me to review this PR if you have free time?
Thank you.

**Comment by klion26** [comment]
@RocMarshal thanks for your contribution. the pr is under my radar now, will review it soon.

File: docs/monitoring/debugging_event_time.zh.md [hunk 1]
```diff
@@ -25,33 +25,28 @@ under the License.
...
-Each Task in Flink exposes a metric called `currentInputWatermark` that represents the lowest watermark received
-by this task. This long value represents the "current event time".
-The value is calculated by taking the minimum of all watermarks received by upstream operators. This means that 
-the event time tracked with watermarks is always dominated by the furthest-behind source.
+可以通过 Flink web 界面或[指标系统]({% link monitoring/metrics.zh.md %})访问 task 的 low watermarks。
 
-The low watermark metric is accessible **using the web interface**, by choosing a task in the metric tab,
-and selecting the `<taskNr>.currentInputWatermark` metric. In the new box you'll now be able to see 
-the current low watermark of the task.
+Flink 中的 task 通过调用 `currentInputWatermark` 方法公开一个指标，该指标表示当前 task 所接收到的 the lowest watermark。这个 long 类型值表示“当前事件时间”。该值通过获取上游算子收到的所有 watermarks 的最小值来计算。这意味着用 watermarks 跟踪的事件时间总是由最落后的 source 控制。
```
Comment (klion26) [inline docs/monitoring/debugging_event_time.zh.md:36]: 这里我不完全确定 ”公开一个指标“ 是否有习惯这么描述，因为我听到更多的是 ”暴露一个指标“ 。如果确定这个是一个习惯描述的话，也可以

File: docs/monitoring/debugging_event_time.zh.md [hunk 2]
```diff
@@ -25,33 +25,28 @@ under the License.
...
 
-## Handling Event Time Stragglers
 
-  - Approach 1: Watermark stays late (indicated completeness), windows fire early
-  - Approach 2: Watermark heuristic with maximum lateness, windows accept late data
+<a name="handling-event-time-stragglers"></a>
+
+## 处理散乱的事件时间
+
+  - 方式 1：Watermark 延迟（表明完整性），窗口提前触发
```
Comment (klion26) [inline docs/monitoring/debugging_event_time.zh.md:49]: 感觉这句话读起来有点不太完整的样子 -- 不太明白这句话的意思，不过我暂时没有一个很好的翻译。

**Comment by RocMarshal** [comment]
Hi, @klion26
Thank you so much for your review.
I made some changes based on your suggestions. Please take a look.

**Comment by rmetzger** [comment]
Thanks a lot for the contribution @RocMarshal and the review @klion26 !

I'm merging the change now.

**Pull request closed without merging**